x-pack/plugins/enterprise_search/public/applications/shared/decode_cloud_id/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
Same as below.
To improve typescript performance, we don't do barrel files anymore. this should be deleted and imports pointing to here should be adjusted.